backend/src/test/java/gov/cdc/usds/simplereport/service/BaseServiceTest.java

```diff
@@ -56,6 +56,7 @@ public void resetOkta() {
   }
 
   protected void initSampleData() {
+    _dataFactory.createValidOrg("DataLorg", "DAT_ORG");
```

Comment:
`DataLorg` again?

Reply:
also, is there a compelling case for... just creating `DAT_ORG` inside `_initService.initAll()`?

Reply:
Yes, I think there is—I didn't want to muck with the org init section if I didn't absolutely have to, so I slapped these two bandaids on. I'd significantly rather have the init code understand multiple organizations, but I think we both agreed that refactoring that while we both had authorization-related branches in flight was a _terrible_ plan, so I have not attempted it.

Reply:
dear reader: i have since configured `initAll()` to create multiple orgs.